File: Selenium/Activities/Activity2.py
from selenium import webdriver
from selenium.webdriver.common.by import By
from webdriver_manager.firefox import GeckoDriverManager
from selenium.webdriver.firefox.service import Service as FirefoxService

serv = FirefoxService(GeckoDriverManager().install())

with webdriver.Firefox(service=serv) as driver:
    driver.get("https://training-support.net/selenium/login-form")
    print("Title of page: ", driver.title)
    driver.find_element(By.ID, "username").send_keys("admin")
    driver.find_element(By.ID, "password").send_keys("password")
    driver.find_element(By.XPATH, "//button[@type='submit']").click()
    print("Text message: ", driver.find_element(By.ID, "action-confirmation").text)
    driver.quit()

File: Selenium/Activities/Activity2.java
package Activities;
import io.github.bonigarcia.wdm.WebDriverManager;
import org.openqa.selenium.*;
import org.openqa.selenium.firefox.FirefoxDriver;


public class Activity2 {
    public static void main(String[] args) throws InterruptedException
    {
        WebDriverManager.firefoxdriver().setup();
        System.setProperty(FirefoxDriver.SystemProperty.BROWSER_LOGFILE,"/dev/null");
        WebDriver newDriver = new FirefoxDriver();
        newDriver.get("https://training-support.net/selenium/login-form");
        System.out.println("Title of page is: "+ newDriver.getTitle());
        newDriver.findElement(By.id("username")).sendKeys("admin");
        newDriver.findElement(By.id("password")).sendKeys("password");
        newDriver.findElement(By.xpath("//button[@type='submit']")).click();
        Thread.sleep(200);
        newDriver.quit();
    }
}
